<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CExtensionBaseDialog</class>
 <widget class="QDialog" name="CExtensionBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Extension Description</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="m_groupboxextensionheader">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="m_textfunctioncount">
          <property name="text">
           <string>Number of Functions</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_editfunctioncount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer2_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>110</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="m_textconstantcount">
          <property name="text">
           <string>Number of Constants</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_editconstantcount">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>463</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="m_textselected">
          <property name="text">
           <string>Selected</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_radioselected">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="2">
       <layout class="QGridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="m_textextensionname_2">
          <property name="text">
           <string>String to search for:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_editextensionname">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="m_textextensionname">
          <property name="text">
           <string>Extension Name</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="m_combobox">
          <property name="autoCompletion">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_textextensionname_3">
        <property name="text">
         <string>Text to find:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QRadioButton" name="m_radiosearchforwards">
        <property name="text">
         <string>Search Forwards</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_editfindtext">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m_pushbuttonfind">
        <property name="text">
         <string>Find again</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="m_radiocasesensitive">
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="m_radiowholewords">
        <property name="text">
         <string>Whole words only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tabwidget">
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>Extension Specification</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="m_textedit">
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Extension Functions</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="CExtensionFunctionListView" name="m_listfunction"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Extension Constants</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="CExtensionConstantListView" name="m_listconstant"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pushbuttonprevious">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonnext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonremove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonadd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>CExtensionConstantListView</class>
   <extends>QWidget</extends>
   <header>extconstantlistwidget/extconstantlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CExtensionFunctionListView</class>
   <extends>QWidget</extends>
   <header>extfunctionlistwidget/extfunctionlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">extfunctionlistwidget/extfunctionlistwidget.h</include>
  <include location="local">extconstantlistwidget/extconstantlistwidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_pushbuttonprevious</sender>
   <signal>clicked()</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>pushbuttonprevious_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonnext</sender>
   <signal>clicked()</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>pushbuttonnext_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonremove</sender>
   <signal>clicked()</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>pushbuttonremove_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonadd</sender>
   <signal>clicked()</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>pushbuttonadd_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonok</sender>
   <signal>clicked()</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>pushbuttonok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_editextensionname</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>editextensionname_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_combobox</sender>
   <signal>activated(QString)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>combobox_activated(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_editfindtext</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>editfindtext_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonfind</sender>
   <signal>clicked()</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>pushbuttonfind_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radioselected</sender>
   <signal>toggled(bool)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>radioselected_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiocasesensitive</sender>
   <signal>toggled(bool)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>radiocasesensitive_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiowholewords</sender>
   <signal>toggled(bool)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>radiowholewords_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radiosearchforwards</sender>
   <signal>toggled(bool)</signal>
   <receiver>CExtensionBaseDialog</receiver>
   <slot>radiosearchforwards_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
